apps/api/src/applicant/external-api.service.ts

```diff
@@ -450,6 +450,7 @@ export class ExternalAPIService {
         assigned_to: { id: string; name: string }[] | undefined;
         registration_date: string;
         applicant_id: string;
+        ats1_id: number;
```

Comment:
Does this need to be number or something? Could this just be a string and then no need to use toString() below?

Reply:
no they send it to us as a `number`, I typically like to use `strings` for anything that doesn't need any type of computation